pkg/cmd/server/kubernetes/master.go

```diff
@@ -103,9 +103,11 @@ func (c *MasterConfig) RunPersistentVolumeProvisioner(client *client.Client) {
 	}
 }
 
-func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client, serviceAccountName string, namespace string) {
 	uid := int64(0)
 	defaultScrubPod := volume.NewPersistentVolumeRecyclerPodTemplate()
+	defaultScrubPod.Namespace = namespace
+	defaultScrubPod.Spec.ServiceAccountName = serviceAccountName
```

Comment:
The controller is running as the service account, so it has permission whether or not the pod's service account does. I don't really want the recycler pod having an API token that lets it do all the things the controller can do